app/containers/Vote.jsx

```diff
@@ -88,4 +82,13 @@ function mapStateToProps(state) {
 
 // Read more about where to place `connect` here:
 // https://github.com/rackt/react-redux/issues/75#issuecomment-135436563
-export default connect(mapStateToProps)(Vote);
+
+const Vote = connect(mapStateToProps)(VoteContainer);
+
+//Data that needs to be called before rendering the component
+//This is used for server side rending via the fetchComponentDataBeforeRender() method
+//Moved the static 'need' to connect wrapper, so initial render doesn't duplicate calls 
+Vote.need =[
```

Comment:
We can safely move this guy back into the Component, given that https://github.com/choonkending/react-webpack-node/pull/201 has been merged 👍